charts/passbolt/templates/deployment.yaml

```
@@ -0,0 +1,107 @@
+      {{- end }}
+      securityContext:
+        {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      containers:
+        - name: {{ .Chart.Name }}
+          securityContext:
+            {{- toYaml .Values.securityContext | nindent 12 }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          ports:
+            - name: http
+              containerPort: 80
+              protocol: TCP
+          env:
+            {{- range .Values.secretsEnv }}
+              {{- $secretName := .secretName }}
+              {{- range $key, $value :=  .env }}
+            - name: {{ $key }}
+              valueFrom:
+                secretKeyRef:
+                  name: {{ $secretName }}
+                  key: {{ $value }}
+              {{- end }}
+            {{- end }}
+            {{- range $key, $value := .Values.env }}
+            - name: "{{ $key }}"
+              value: "{{ $value }}"
+            {{- end }}
+            - name: APP_FULL_BASE_URL
+              value: https://{{ .Values.passbolt.baseUrl }}
+          volumeMounts:
+            {{- if .Values.persistence.enabled }}
+            - name: data
+              mountPath: /var/www/passbolt/config/gpg
+              subPath: gpg
+            - name : data
+              mountPath : /var/www/passbolt/webroot/img/public/Avatar
+              subPath: avatar
+            {{- end }}
+            {{- if .Values.passbolt.pro }}
+            - name: license
+              mountPath: /var/www/passbolt/config/license
+              subPath: subscription_key.txt
+              readOnly: true
+            {{- end }}
+          livenessProbe:
+            httpGet:
+              path: /healthcheck/status.json
+              port: http
+            initialDelaySeconds: 10
+            timeoutSeconds: 10
+          readinessProbe:
+            httpGet:
+              path: /healthcheck/status.json
+              port: http
+            initialDelaySeconds: 20
+            timeoutSeconds: 10
+          resources:
+            {{- toYaml .Values.resources | nindent 12 }}
+      volumes:
+        {{- if .Values.passbolt.pro }}
+        - name: license
+          secret:
+            secretName: {{ template "passbolt.name" . }}-key
+        {{- end }}
+        - name: data
+          {{- if .Values.persistence.enabled }}
+          persistentVolumeClaim:
+            claimName: {{ .Values.persistence.existingClaim | default (include "passbolt.fullname" .) }}
+          {{- else }}
+          emptyDir: {}
+          {{- end -}}
+      {{- with .Values.nodeSelector }}
+      nodeSelector:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      {{- with .Values.affinity }}
+      affinity:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      {{- with .Values.tolerations }}
+      tolerations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
```

Ajout retour à la ligne sur `deployment.yaml`
```suggestion
      {{- end }}

```